storage/id_alloc.go

```diff
@@ -100,14 +100,13 @@ func (ia *idAllocator) start() {
 					res client.KeyValue
 				)
 				for r := retry.Start(idAllocationRetryOpts); r.Next(); {
-					task := ia.stopper.StartTask()
-					if !task.Ok() {
+					var idKey proto.Key
+					if !ia.stopper.RunTask(func() {
+						idKey = ia.idKey.Load().(proto.Key)
```

Comment:
this doesn't need to be inside the `RunTask`, which should save the `var` declaration above